src/components/typography/text/README.md

```diff
@@ -26,6 +26,7 @@ Wraps the given text in the given HTML header `size`.
 | `children`    | `PropTypes.node` |    ✅    | -                    | -       | -                                                              |
 | `title`       | `String`         |    -     | -                    | -       | Text to show in a tooltip on hover over the element            |
 | `truncate`    | `Bool`           |    -     | -                    | `false` | Option for truncate content in case the screen has small width |
+| `dataTest`    | `String`         |    -     | -                    | -       | Prop for E2E testing                                           |
```

Comment:
instead of making a new prop, can you follow how it's done [here](https://github.com/commercetools/ui-kit/tree/master/src/components/inputs/text-input) in text-input?

Don't add it to the table instead add 
```
The component further forwards all data- attributes to the underlying component.
```